### PR TITLE
feat: add apkanalyzer

### PR DESCRIPTION
**Pull request by KazuCocoa**
```
info AppiumDoctor  ✔ JAVA_HOME is set to: /Library/Java/JavaVirtualMachines/jdk-9.0.4.jdk/Contents/Home
info AppiumDoctor    Checking adb, android, emulator, apksigner.jar, apkanalyzer
info AppiumDoctor      'adb' is in /Users/kazuaki/Library/Android/sdk/platform-tools/adb
info AppiumDoctor      'android' is in /Users/kazuaki/Library/Android/sdk/tools/android
info AppiumDoctor      'emulator' is in /Users/kazuaki/Library/Android/sdk/emulator/emulator
info AppiumDoctor      'apkanalyzer' is in /Users/kazuaki/Library/Android/sdk/cmdline-tools/latest/bin/apkanalyzer
```

### CODE REVIEW

**Review by mykola-mokhnach — CHANGES_REQUESTED** [review]
I assume on Windows the script has .bat extension

**Comment by KazuCocoa** [comment]
let me check

**Comment by KazuCocoa** [comment]
image

File: lib/android.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ class JavaOnPathCheck extends DoctorCheck {
 class AndroidToolCheck extends DoctorCheck {
   constructor () {
     super();
-    this.tools = ['adb', 'android', 'emulator'];
+    this.tools = ['adb', 'android', 'emulator', system.isWindows() ? 'apkanalyzer.bat' : 'apkanalyzer'];
```
Comment (mykola-mokhnach) [inline lib/android.js:39]:
```
`apkanalyzer${system.isWindows() ? '.bat' : ''}`
```

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**